### ISSUE TITLE
No OpenShift version available, please ensure your systems are fully registered and have access to appropriate yum repositories

### ISSUE THREAD

**Issue by tobmat** [dwmkerr/terraform-aws-openshift#13]
I changed the deployment_type=openshift-enterprise
I manually logged into the master,node1, and node2 and registered with redhat and attached a pool ID with openshift subscriptions.

Is there a manual way for me to verify I have the subscription I need?  Here are the ones associated with the pool ID I used:

Red Hat OpenShift Container Platform Broker/Master Infrastructure

Red Hat OpenShift Enterprise Infrastructure
Red Hat OpenShift Container Platform
Red Hat OpenShift Enterprise Client Tools

Any other ideas on what my issue may be?

Thanks!

Note - I was able to successfully run this with deployment type origin

**Comment by dwmkerr** [comment]
Hi @tobmat hmm that's a good question. I initially set up Enterprise and had some similar problems, but the issue is that I don't have an enterprise license anymore to test the issue. I had tonnes of problems registering my RHEL instances and had to call them from support, it turns out my license didn't allow me to install on the required three nodes.

As it's an enterprise it might be worth checking with their support, there's little I can do without a license as everything seems to change quite a bit when you install enterprise, sorry about that!

**Comment by tobmat** [comment]
Thanks for the info.  I will open a support ticket with them.  For contest is this process performing the unattended install as referenced here:  https://docs.openshift.com/enterprise/3.0/install_config/install/quick_install.html#running-an-unattended-installation

If so,  is the installer.cfg.yml file referenced dynamically created or located somewhere in the repo.  I tried to figure it out but sort of got lost :-).  Any direction would be appreciated.  I will let you know whatever I find out from redhat.  Thanks!

**Comment by jayunit100** [comment]
@dwmkerr we are happy to test this for you on a regular basis,  with OSE , have time to sync over the phone about it to collaborate ?

**Comment by dwmkerr** [comment]
@jayunit100 yes that would be helpful if I can get help testing! I would be able to talk the week after next (am swamped with work at the moment) would that work for you?

**Comment by tobmat** [comment]
My priorities have been shifted to some other initiatives at the moment.  I'm pretty sure we will return to this at some point.

**Comment by jayunit100** [comment]
Yeah that's fine; chat next week about this
